docs/toc.json

```diff
@@ -2,32 +2,32 @@
     "guides": [{
         "title": "Authentication",
         "id": "authentication",
-        "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/authentication/readme.md",
+        "edit": "https://github.com/googleapis/gcloud-common/edit/master/authentication/readme.md",
```

Comment:
I'm not sure when `gcloud-common` will move to `googleapis` (it's not migrated yet). Maybe leave them as they are for now?

Reply:
It's actually already been moved, but thanks for catching it because i need to replace `gcloud-common` with `google-cloud-common`! https://github.com/googleapis/google-cloud-common

Reply:
You're right! It already moved :)